doc/docs/api/examples.md

```diff
@@ -1,21 +1,46 @@
 <!-- markdownlint-disable-file MD013 MD024 -->
 # Examples using the public D-Bus API
 
-## GoLang
+These steps were based on CentOS Stream 9
 
-First, make sure the system has golang installed and download the dbus library.
+## Ruby
 
-These steps were based on CentOS Stream 9:
+### Installing ruby and dbus module
 
-### Installing golang
+``` bash
+dnf install ruby -y
+gem install ruby-dbus
+```
+
+### Build the source code
```

Comment:
Isn't this section more like `Build and run the example code`?

Reply:
sure, works for me.